resources/warnings.json

```diff
@@ -17428,8 +17428,8 @@
     "url": "https://www.jenkins.io/security/advisory/2023-12-13/#SECURITY-3168",
     "versions": [
       {
-        "lastVersion": "2.6",
-        "pattern": ".*"
+        "lastVersion": "4.228.v0c3e8682ff1f",
+        "pattern": "[1-3][.][0-9]|4[.]22[0-8][.]v[a-f0-9_]+(|[.-].+)"
```

Comment:
Can be simplified to

```suggestion
        "pattern": "([1-3]|4[.]22[0-8])(|[.-].+)"
```

Reply:
Ok for me. I was not sure the pattern was matching from the beginning of the string.